doc/release-notes-bucash.md

```diff
@@ -29,189 +30,352 @@ shut down (which might take a few minutes for older versions), then run the
 installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
 bitcoind/bitcoin-qt (on Linux).
 
-If you are upgrading from a release older than 1.1.2.0, your UTXO database will be converted
-to a new format. This step could take a variable amount of time that will depend
-on the performance of the hardware you are using.
-
-Other than that upgrading from a version lower than 1.5.0.0 your client is probably stuck
+Upgrading from a version lower than 1.6.0.0 would probably lead your client to be stuck
 on a minority chain and need some manual intervention to make so that it follow the majority
 chain after the upgrade. For more detail please look at `reconsidermostworkchain` RPC commands.
 
-Main Changes in 1.6.0.1
+Main Changes in 1.7.0
 ---------------------
 
 This is list of the main changes that have been merged in this release:
 
-- Improve Graphene decode rates by padding IBLT
+- Mempool synchronization via Graphene primitives
+- Intelligent unconfirmed transaction forwarding
+- Child-Pay-For-Parent implementation based on Ancestor Grouped Transactions (AGT)
+- Graphene ver 2.1 and IBLT specifications
```

Comment:
I feel like we need a few more words here for Child Pays for Parent, such as, it can also efficiently mine long unconfirmed chains.

Reply:
never mind, i see you added that below